include/z64scene.h

```diff
@@ -417,4 +419,139 @@ typedef enum {
+    /* 0x08 */ SCENECMD_ID_ROOM_BEHAVIOR,
+    /* 0x09 */ SCENECMD_ID_UNUSED_9,
+    /* 0x0A */ SCENECMD_ID_MESH,
+    /* 0x0B */ SCENECMD_ID_OBJECT_LIST,
+    /* 0x0C */ SCENECMD_ID_POS_LIGHT_LIST,
+    /* 0x0D */ SCENECMD_ID_PATH_LIST,
+    /* 0x0E */ SCENECMD_ID_TRANSI_ACTOR_LIST,
+    /* 0x0F */ SCENECMD_ID_LIGHT_LIST,
+    /* 0x10 */ SCENECMD_ID_TIME_SETTINGS,
+    /* 0x11 */ SCENECMD_ID_SKYBOX_SETTINGS,
+    /* 0x12 */ SCENECMD_ID_SKYBOX_DISABLES,
+    /* 0x13 */ SCENECMD_ID_EXIT_LIST,
+    /* 0x14 */ SCENECMD_ID_END,
+    /* 0x15 */ SCENECMD_ID_SOUND_SETTINGS,
+    /* 0x16 */ SCENECMD_ID_ECHO_SETTINGS,
+    /* 0x17 */ SCENECMD_ID_CUTSCENE_LIST,
+    /* 0x18 */ SCENECMD_ID_ALTERNATE_HEADER_LIST,
+    /* 0x19 */ SCENECMD_ID_MISC_SETTINGS,
+    /* 0x1A */ SCENECMD_ID_TEXTURE_ANIM_LIST,
+    /* 0x1B */ SCENECMD_ID_ACTOR_CUTSCENE_LIST,
+    /* 0x1C */ SCENECMD_ID_MINIMAP_INFO,
+    /* 0x1D */ SCENECMD_ID_UNUSED_1D,
+    /* 0x1E */ SCENECMD_ID_MINIMAP_COMPASS_ICON_INFO
+} SceneCommandTypeID;
+
+#define SCENECMD_SPAWN_LIST(spawnNum, spawnList)                 \
+    { SCENECMD_ID_SPAWN_LIST, spawnNum, CMD_PTR(spawnList) }
+
+#define SCENECMD_ACTOR_LIST(actorNum, actorList)                 \
+    { SCENECMD_ID_ACTOR_LIST, actorNum, CMD_PTR(actorList) }
+
+#define SCENECMD_CAM_LIST(camNum, camList)       \
+    { SCENECMD_ID_ACTOR_CUTSCENE_CAM_LIST, camNum, CMD_PTR(camList) }
+
+#define SCENECMD_COL_HEADER(colHeader)                    \
+    { SCENECMD_ID_COL_HEADER, 0, CMD_PTR(colHeader) }
+
+#define SCENECMD_ROOM_LIST(roomNum, roomList)                 \
+    { SCENECMD_ID_ROOM_LIST, roomNum, CMD_PTR(roomList) }
+
+#define SCENECMD_WIND_SETTINGS(xDir, yDir, zDir, strength)                     \
+    { SCENECMD_ID_WIND_SETTINGS, 0, CMD_BBBB(xDir, yDir, zDir, strength) }
+
+#define SCENECMD_ENTRANCE_LIST(entranceList)                    \
+    { SCENECMD_ID_ENTRANCE_LIST, 0, CMD_PTR(entranceList) }
+
+#define SCENECMD_SPECIAL_FILES(elfMessageFile, specialObject)               \
+    { SCENECMD_ID_SPECIAL_FILES, elfMessageFile, CMD_W(specialObject) }
+
+#define SCENECMD_ROOM_BEHAVIOR(currRoomUnk3, currRoomUnk2, currRoomUnk5, msgCtxunk12044, enablePosLights,  \
+                                kankyoContextUnkE2)                                                         \
+    {                                                                                                       \
+        SCENECMD_ID_ROOM_BEHAVIOR, currRoomUnk3,                                                           \
+            currRoomUnk2 | _SHIFTL(currRoomUnk5, 8, 1) | _SHIFTL(msgCtxunk12044, 10, 1) |                   \
+                _SHIFTL(enablePosLights, 11, 1) | _SHIFTL(kankyoContextUnkE2, 12, 1)                        \
+    }
+
+#define SCENECMD_UNUSED_9()                   \
+    { SCENECMD_ID_UNUSED_9, 0, CMD_W(0) }
+
+#define SCENECMD_MESH(meshHeader)                    \
+    { SCENECMD_ID_MESH, 0, CMD_PTR(meshHeader) }
+
```

Comment:
While we're talking about names, I've never really liked the term "mesh" for this. Its pretty a pretty loose term and can be referencing just the geometry that makes up the maps, when really this is the whole display list. `DLISTS` or something would make more sense to me, but curious what others think

Reply:
DLISTS imo is too technical, I'm find with MODEL though.

Reply:
What about `POLYGON_TYPE`? The `meshHeader` ptr points to a `PolygonType0`, `PolygonType1` or `PolygonType2`

Reply:
its more than just that, it points to the display lists for the whole room. I dont think thats too technical, its just describing what it is.

I dont like model for the same reason as mesh, model sounds like just the 3d geometry when its the whole thing, materials and all.

If you dont want to use the word dlists can maybe go with simply GFX. but we use dlist to reference the same thing everywhere else, so idk

Reply:
If that's the case, then `DLISTS` sounds good to me. But I would like to hear if anybody else has something to say about this

Reply:
Model or mesh is pretty obvious what it does though, even if there is a 2% chance that someone might see it as something else, I think it's more clear to people who don't know all the technical details about the game etc like what a displaylist is